doc/01_specification/index.rst

```diff
@@ -716,13 +736,15 @@ integrated in the continuous integration flow.
 |        | CV32E20\_?\_fpga configuration on Kintex 7 FPGA technology. |
 |        |                                                             |
 |        | Metric details to be supplied later.                        |
+.. TODO PPA-30 Clock frequency requirement on a FPGA implementation
 +--------+-------------------------------------------------------------+
 | PPA-50 | CV32E20 should operate at more than ? MHz in the            |
 |        | CV32E20\_? configuration on 16-nm FFT technology in the     |
 |        | worst-case frequency corner with the fastest threshold      |
 |        | voltage.                                                    |
 |        |                                                             |
 |        | Metric details to be supplied later.                        |
+.. TODO PPA-50 Clock frequency requirement on an (16nm FinFET) ASIC implementation
```

Comment:
I would recommend removing PPA-50 altogether.  We currently do not have anyone pushing a physical implementation of this device and certainly not for a 16nm tech-node.  Leave in the `TODO` comment.